### ISSUE TITLE
Vanilla Armour Expanded Patch --- has gamebreaking change

### ISSUE THREAD

**Issue by brightsideguy** [AthenaSulisMinerva/CombatExtendedFastTrack#291]
For quilted vests, the material effect was put on an insanely high level (12.5), making the quilted vest better than Power Armor if using certain mod materials. 

Also, the reasoning for this change (written in the code) don't make sense for what the code actually does.

<StuffEffectMultiplierArmor>12.5</StuffEffectMultiplierArmor>

**Comment by AthenaSulisMinerva** [comment]
@brightsideguy, please follow the issue reporting guidelines and use the proper bug reporting template to report this issue - otherwise, I will close this as invalid.

**Comment by Wuldahfel** [comment]
I think the problem here is not the Quilted Vest itself, but the leathers whatever mod you have adds. My quilted vests are perfectly fine, a normal quality elephant leather Quilted Vest has 0.63 mm RHA sharp and 0.5 MPa blunt. Nowhere near even a poor quality Power Armor (15 mm RHA sharp, 33.75 MPa blunt).

**Comment by brightsideguy** [comment]
@Wuldahfel  You're right, the leathers are the main culprit. This is from Alpha Animals, however. So that means the FastTrack patch of Alpha Animals needs some balancing for Alpha Animals and Vanilla Armour Expanded to make sense together.    

But maybe that's outside of the work Fast Track wants to be involved with?

**Comment by brightsideguy** [comment]
@AthenaSulisMinerva since this isn't a bug but rather an issue of balance, is there a different arena I should be posting this issue?  Or do you want me to just fill in the bug report as closely as I can for it to describe a balance issue?

**Comment by NickZiggy** [comment]
> @AthenaSulisMinerva since this isn't a bug but rather an issue of balance, is there a different arena I should be posting this issue? Or do you want me to just fill in the bug report as closely as I can for it to describe a balance issue?

He has been deflecting issues like this for a while now. It's sad that FT took the same turn as the original CE.
